airflow/providers/amazon/aws/example_dags/example_ecs_fargate.py

```diff
@@ -64,8 +65,8 @@
     },
     network_configuration={
         "awsvpcConfiguration": {
-            "securityGroups": ["sg-123abc"],
-            "subnets": ["subnet-123456ab"],
+            "securityGroups": [os.environ.get("SECURITY_GROUP_ID")],  # or simply ["sg-123abc"],
+            "subnets": [os.environ.get("SUBNET_ID")],  # or simply ["subnet-123456ab"],
```

Comment:
You can pass fall-back values to `.get`
```suggestion
            "securityGroups": [os.environ.get("SECURITY_GROUP_ID", "sg-123abc")],
            "subnets": [os.environ.get("SUBNET_ID", "subnet-123456ab")],
```

Reply:
these are example (placeholder) values, because this is example dag. I just want to show that the reader can use hard-coded values. Also, at the same time the example dag can be used in system tests. I can change as you suggest but the "fallback values" are not going to work.